src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -1,17 +1,16 @@
+import { FC, useContext } from 'react';
 import {
     Draggable,
     DraggingStyle,
     NotDraggingStyle,
 } from 'react-beautiful-dnd';
-import { FC, useContext } from 'react';
 import { draggableProps } from '@/interfaces/availabilityPanel.type';
 import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
+import { DUMMY_PROFILE } from '@/components/constants/display-sections';
```

Comment:
Use alias name `placeholderImageURL` for `DUMMY_PROFILE`

Snippet:

```js
import { DUMMY_PROFILE as placeHolderImageURL } from '@/components/constants/display-sections';
```

Reply:
Changes done